cmd/nerdctl/container_inspect.go

```diff
@@ -30,51 +30,43 @@ import (
 	"github.com/docker/cli/templates"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"github.com/urfave/cli/v2"
+	"github.com/spf13/cobra"
 )
 
-var containerInspectCommand = &cli.Command{
-	Name:         "inspect",
-	Usage:        "Display detailed information on one or more containers.",
-	ArgsUsage:    "[flags] CONTAINER [CONTAINER, ...]",
-	Description:  "Hint: set `--mode=native` for showing the full output",
-	Action:       ContainerInspectAction,
-	BashComplete: containerInspectBashComplete,
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "mode",
-			Usage: "Inspect mode, \"dockercompat\" for Docker-compatible output, \"native\" for containerd-native output",
-			Value: "dockercompat",
-		},
-		&cli.StringFlag{
-			Name:    "format",
-			Aliases: []string{"f"},
-			Usage:   "Format the output using the given Go template, e.g, '{{json .}}'",
-		},
-	},
+func newContainerInspectCommand() *cobra.Command {
+	var containerInspectCommand = &cobra.Command{
+		Use:               "inspect [flags] CONTAINER [CONTAINER, ...]",
+		Short:             "Display detailed information on one or more containers.",
+		Long:              "Hint: set `--mode=native` for showing the full output",
+		RunE:              containerInspectAction,
+		ValidArgsFunction: containerInspectShellComplete,
+	}
+	containerInspectCommand.Flags().String("mode", "dockercompat", `Inspect mode, "dockercompat" for Docker-compatible output, "native" for containerd-native output`)
+	containerInspectCommand.Flags().StringP("format", "f", "", "Format the output using the given Go template, e.g, '{{json .}}'")
+	return containerInspectCommand
 }
 
-func ContainerInspectAction(clicontext *cli.Context) error {
-	if clicontext.NArg() == 0 {
+func containerInspectAction(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
 		return errors.Errorf("requires at least 1 argument")
 	}
 
-	client, ctx, cancel, err := newClient(clicontext)
+	client, ctx, cancel, err := newClient(cmd)
 	if err != nil {
 		return err
 	}
 	defer cancel()
 
 	f := &containerInspector{
-		mode: clicontext.String("mode"),
+		mode: "dockercompat",
```

Comment:
cmd.Flags().GetString("mode") because we have two mode : `native`  and `dockercompat`